### ISSUE TITLE
WIFI & ETHERNET metric

### ISSUE THREAD

**Issue by bdbo** [MicrosoftDocs/windowsserverdocs#1527]
We see 3 issues so far with split tunneling:

- The VPN adapter metric is dynamic "Get-NetIPInterface" shows that it is sometimes before or after the wifi/ethernet connection. So sometimes  all DNS is resolved by the VPN, sometimes with the ISP. Highest VPN metric was 55.

- DnsSuffix  XML setting is microsoft only? It doesn't seem to work with Chrome/Firefox. Also FQDN suffix is not added to resolve netbios names.

-  Smart Name Resolution can cause DNS resolution on the wrong DNS ip.

Here you can force the adapter metric:
%userprofile%\AppData\Roaming\Microsoft\Network\Connections\Pbk\rasphone.pbk file and changing the lines IpInterfaceMetric and Ipv6InterfaceMetric
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e7453068-95eb-2a42-fd0b-553ae593adb0
* Version Independent ID: b39b2cb7-25b1-2af3-4954-dbcc04c836e9
* Content: [Configure Windows 10 Client Always On VPN Connections](https://docs.microsoft.com/en-us/windows-server/remote/remote-access/vpn/always-on-vpn/deploy/vpn-deploy-client-vpn-connections)
* Content Source: [WindowsServerDocs/remote/remote-access/vpn/always-on-vpn/deploy/vpn-deploy-client-vpn-connections.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/remote/remote-access/vpn/always-on-vpn/deploy/vpn-deploy-client-vpn-connections.md)
* Product: **windows-server-threshold**
* GitHub Login: @shortpatti
* Microsoft Alias: **pashort**

**Comment by shortpatti** [comment]
@MihaiSP, can you take a look at this issue?

**Comment by mghrivera** [comment]
I have run into the same issue. Where I specified a route in the ProfileXML. 
for some odd reason the connection doesn't use DNS for lookup and tries to send the traffic to the VPN instead of the outside split tunneling. issue number #1931
the issue here is when I do the route it sets interface metric of the route of 35 which is lower than the wireless and vpn connection. so when I do lookup for outside work resources it actually trying to use vpn dns which resolved the ip to the private ip address and try to send the traffic over VPN. setting the metric on the wireless interface lower than the route interface metric everything works fine.

**Comment by bdbo** [comment]
What we saw, when you create a Windows 2012r2 server GPO that contains a "Network\DNS Client" setting like "DNS suffix search list", this GPO (Registry.Pol) also creates an empty subfolder "DnsPolicyConfig". So on the windows 10 DNSClient folder is created but also this reg key is added,:

HKLM:\Software\Policies\Microsoft\Windows NT\DNSClient\DnsPolicyConfig

This happend when we still had the DirectAccess servers installed.

This empty key "DnsPolicyConfig" breaks Always On VPN NRPT on the windows 10 client, Get-DNSClientNrptRule returns the correct DomainNameInformation, but Get-DNSClientNrptPolicy will be empty. Manually deleting the key "DnsPolicyConfig" fixes the issue, next GPO refresh, the issue returns.

Basically an empty/existing"DnsPolicyConfig" reg key overrules the NRPT policy in the VPN XML file. Removing DirectAccess configuration & re-edit something in the "Network\DNS Client" & save GPO   resolved the issue.

**Comment by nschindler** [comment]
I have the same issue with the DnsPolicyConfig key persisting.  Even after making a change to a Network\DNS Client policy the key was not removed.  I also tried creating and removing an NRPT rule.  
In my case, unfortunately, it's coming from the Default Domain Policy according to Policy Analyzer.  
Anyone have any ideas other than dcgpofix?  I'd consider that a last resort.

**Comment by TonyBunce** [comment]
We ran into this same issue and we were able to use powershell to modify the GPO and remove the empty registry key.  The "Remove-GPRegistryValue" can't delete a null key, so I set it to an empty string and was then able to remove it.

`Set-GPRegistryValue -Name "<Name of GPO>" -Key "HKLM\SOFTWARE\Policies\Microsoft\Windows NT\DNSClient\DnsPolicyConfig" -Type String -Value ""
Remove-GPRegistryValue -Name "<Name of GPO>" -Key "HKLM\SOFTWARE\Policies\Microsoft\Windows NT\DNSClient\DnsPolicyConfig"
`

**Comment by Dave-Russell** [comment]
We have Direct Access and will be progressively migrating to AOVPN. I can confirm that if Direct Access is present, NRPT Rules are skipped. I found an easy solution was to change the name of the Reg Key DnsPolicyConfig to DnsPolicyConfig1 which effectively rendered everything related to the Key. The NRPT Rules worked. Microsoft needs to get on a sort this out so DA can coexist as it was intended to do.

**Comment by shortpatti** [comment]
Can a Microsoft employee PLEASE PLEASE PLEASE remove my name from all the documents I used to own and manage? I signed the divorce papers from Microsoft in Nov 2018 and would LOVE IT if a Microsoft writer would take ownership of the content. I don't work there, and don't get paid to correct these documents anymore. I feel sorry for the IT Professionals using these products because there's no one at Microsoft who really cares about the IT Pro :( It's obvious because these documents haven't been maintained for over 18 months.

PLEASE PLEASE PLEASE someone remove my name from these documents!!!!! I can unsubscribe until my face turns blue, but that's 100s if not 1000s of documents.

**Comment by Dave-Russell** [comment]
Hi, sorry to hear about the situation.
I will escalate this issue to Microsoft.

Thanks,

Dave

**Comment by IngridAtMicrosoft** [comment]
#please-close
